### DISCUSSION TITLE
13:9: Empty Subsection Name - How do you fix this?

### DISCUSSION THREAD

**Discussion by cswoods6** [jesseduffield/lazygit#1770] [Q&A]
Been stuck with this error for a few days now. I am not sure how I got the error, or how to fix the error. I tried replacing the config file which didn't do anything. Any help is appreciated.
![Image 2-14-2022 at 7 15 AM](https://user-images.githubusercontent.com/42786886/153871112-b3522950-1ed5-420f-b2aa-4f1df6738ef1.jpg)

**Comment by mark2185** [comment]
Does this still occur? Can it be reproduced with current `master`?

**Reply by cswoods6** [reply]
I am not able to recreate the issue any longer. For various reasons I switched away from codespaces and back to developing locally. Setting up the project again fixed the issue
